dashboard/api_test.py

```diff
@@ -309,6 +363,15 @@ def setUpTestData(cls):
             enr_end=now+timedelta(weeks=1),
             edx_key="course-v1:MITx+8.MechCX+2014_T1"
         )
+        # and a run that is past and verified
```

Comment:
Would we ever have a certificate that's for a course run that has not reached its end date yet? Maybe the course gave our grades early? If you think that's a possibility can you add a test case for it?